ShopNow/UserApp/views.py

```diff
@@ -41,40 +54,59 @@ def registerUser(request):
                 serialized.save()
                 # print(f"""Username: {serialized.data.get('username')}, Email: {serialized.data.get('useremail')},
                 #     Password: {serialized.data.get('password')}, Confirm Password: {serialized.data.get('confirm_password')}""")
-                return Response(serialized.data)
+                return Response(serialized.data,status=status.HTTP_201_CREATED)
             else:
-                return Response("your data is invalid")
+                return Response(f"your data is invalid,{serialized.errors}",status=status.HTTP_400_BAD_REQUEST)
         else:
-            return Response("Data is not inluded in request.")
+            return Response("Data is not inluded in request.",status=status.HTTP_400_BAD_REQUEST)
     else:
-        return Response("Invalid method")
+        return Response("Invalid method",status=status.HTTP_400_BAD_REQUEST)
 
 
        
-@csrf_exempt
+
 @api_view(['POST'])
 def loginUser(request):
     if request.method=='POST':
         if request.data:
             serialized=loginSerializer(data=request.data)
             print(request.data)
             if serialized.is_valid():
```

Comment:
Its fine but to make code more readable i believe if we check the serailizer validity like 

```
if not serialized.is_valid():
    return Response(serialized.errors, status=status.HTTP_400_BAD_REQUEST)

```

this check would remove and all code come free from check.